nornir_table_inventory/plugins/inventory/table.py

```diff
@@ -134,7 +140,8 @@ def load(self) -> Inventory:
         hosts = Hosts()
 
         for host_dict in self.hosts_list:
-            hosts[host_dict['name']] = _get_host_obj(host_dict)
+            if not _empty(host_dict['name']):
```

Comment:
In Nornir ，name must not be None. I think ,we should follow the raw rule. I know your thinking，this is for switch stacks or chasis switch such as Cisco nexus 2000. But I think ，Automation think over NOS（Network Operate System） more than Physical Switch. In our CMDB ,they are two model. I get data from NOS model., and parse data from NOS to NOS and physical switch.
And in our DC , every swith with a chasis must have a name.

Reply:
That's the point! This part of the code will ignore rows with an empty name. But it still continues to parse others.
My original work on this was that you made it mandatory to fill in port field which is definitely not mandatory by Nornir. NaN values caused problem by converting float('nan') to int. So I made as many fields optional as possible.